tools/common/Application.cs

```diff
@@ -64,6 +64,7 @@ public partial class Application
 		public bool? EnableCoopGC;
 		public bool EnableSGenConc;
 		public bool UseInterpreter;
+		public List<string> InterpretedAssemblies = new List<string> ();
```

Comment:
hmm... it's inside `common` which is shared with XM
not sure it's the best place, Maybe under `#if MTOUCH` ?

Reply:
Fixed by moving the code to an mtouch-specific file.